### PR TITLE
mgr/dashboard: Add QA tests for RBD configuration

### PR DESCRIPTION
**Pull request by p-na**
Fixes: http://tracker.ceph.com/issues/37572

Signed-off-by: Patrick Nawracay <pnawracay@suse.com>

- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: qa/tasks/mgr/dashboard/test_pool.py [hunk 1]
```diff
@@ -71,49 +112,27 @@ def _get_pool(self, pool_name):
...
             pool = self._get_pool(pool['pool_name'])
-        for p in [prop, pgp_prop]:  # Should have the same values
+        for p in ['pg_num', pgp_prop]:  # Should have the same values
             self.assertEqual(pool[p], int(value), '{}: {} != {}'.format(p, pool[p], value))
 
     @classmethod
     def tearDownClass(cls):
         super(PoolTest, cls).tearDownClass()
-        for name in ['dashboard_pool1', 'dashboard_pool2', 'dashboard_pool3', 'dashboard_pool_update1']:
+        for name in ['dashboard_pool1', 'dashboard_pool2', 'dashboard_pool3', 'dashboard_pool_update']:
```
Comment (s0nea) [inline qa/tasks/mgr/dashboard/test_pool.py]: I'm wondering if pool `dashboard_pool_update` needs to be deleted here. Where is it created?

Comment (p-na) [reply qa/tasks/mgr/dashboard/test_pool.py]: Thanks! It was a remnant of the previous implementation.

**Comment by s0nea** [comment]
The last QA run failed:
http://pulpito.ceph.com/tdehler-2019-05-09_07:18:40-rados:mgr-wip-tdehler-testing-27774-distro-basic-smithi/

The error is not related I think as it happened on https://github.com/ceph/ceph/pull/27072 as well:
https://tracker.ceph.com/issues/39648

**Comment by p-na** [comment]
jenkins test make check arm64

**Pull request merged**